dash-pipeline/SAI/src/p4meta.cpp

```diff
@@ -0,0 +1,407 @@
+extern "C" {
+#include "saimetadata.h"
+}
+
+#include "utils.h"
+#include "p4meta.h"
+
+using namespace dash;
+using namespace dash::utils;
+
+namespace dash
+{
+    const P4MetaKey* P4MetaTable::get_meta_key(
+            _In_ sai_attr_id_t attr_id) const
+    {
+        for (auto i=0u; i<keys.size(); i++) {
+            if (attr_id == keys[i].attr_id) {
+                return &keys[i];
+            }
+        }
+
+        return nullptr;
+    }
+
+    const P4MetaKey* P4MetaTable::get_meta_key(
+            _In_ const std::string &key_name) const
+    {
+        for (auto i=0u; i<keys.size(); i++) {
+            if (key_name == keys[i].name) {
+                return &keys[i];
+            }
+        }
+
```

Comment:
over entire PR incosistent placemtn { once in end of function once in new line, over all project has { in new line

Reply:
fixed